packages/console/src/pages/ApiResourceDetails/components/DeleteForm/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { toast } from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import TextInput from '@/components/TextInput';
+import Close from '@/icons/Close';
+import api from '@/utilities/api';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  id: string;
+  name: string;
+  onClose: () => void;
+};
+
+const DeleteForm = ({ id, name, onClose }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  const navigate = useNavigate();
+
+  const [inputName, setInputName] = useState<string>('');
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const inputMismatched = inputName !== name;
+
+  const handleDelete = async () => {
+    try {
+      setIsDeleting(true);
+      await api.delete(`/api/resources/${id}`);
+      onClose();
+      navigate(`/api-resources`);
+      toast.success(t('api_resource_details.api_resource_deleted', { name }));
+    } catch (error: unknown) {
+      console.error(error);
```

Comment:
Error should be handled here?  Maybe add a todo if global error toast is not implemented yet?

Reply:
@xiaoyijun `api` contains global error toast, so no need to catch here, unless you need to handle special cases.